### ISSUE TITLE
Badge number still exist when re-install application

### ISSUE THREAD

**Issue by SanjanaTailor** [katzer/cordova-plugin-badge#135]
Hi all,
I installed my app and do something to add notification. There is a badge number on the icon app as I expected. But when I remove the app and then install it again, I still see the badge number on the icon app until I start the app. As my expectation, I think the badge number must be clear when the application is removed, but here, it is not. 
How can I fix that? 
Plugin info
"dependencies": {
"@ionic-native/badge": "^4.20.0",
"cordova-plugin-badge": "0.8.8",
}
ionic info

Ionic:

ionic (Ionic CLI) : 4.1.2 Ionic Framework : ionic-angular 3.8.0 @ionic/app-scripts : 3.0.1

Cordova:

cordova (Cordova CLI) : 9.0.0 (cordova-lib@9.0.1) Cordova Platforms : android 6.3.0 Cordova Plugins : no whitelisted plugins (14 plugins total)

System:

Android SDK Tools : 26.1.1 ( NodeJS : v8.11.2 npm : 6.9.0 OS : Windows 10

**Comment by alidavid0418** [comment]
You could think of force-clear.
```
cordova.plugins.notification.badge.clear();
```

**Comment by SanjanaTailor** [comment]
@alidavid0418 : I am already doing it but this is uninstall the app so by default plugin willl call above function. can it be possible to give an example programmatically

**Comment by rnng** [comment]
I'm facing this issue. Did you found a solution for this, @SanjanaTailor ?

**Comment by SanjanaTailor** [comment]
No @rnng

**Comment by viananicolas** [comment]
Also facing this issue.

**Comment by SanjanaTailor** [comment]
@viananicolas  : Is it for both android and iOS ? After uninstall and install the app shows badge count from cache(old)

**Comment by viananicolas** [comment]
Yeah, it's gone for me by uninstalling version 0.8.8 and installing an older version, like 0.8.6.

**Comment by katzer** [comment]
I cannot reproduce that
